src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx

```diff
@@ -580,7 +580,7 @@ const MeasureGroups = (props: MeasureGroupProps) => {
   const orderingMeasureGroupObservations = (group: Group) => {
```

Comment:
not your ticket but while you are here can we update the method name? we really need to work on naming conventions

Reply:
Any suggestions ? I am having hard time understanding what is even happening here.

Reply:
Sorry I was the one who had given that name, so basically in that function we are ordering the measure observations such that always first denominatorObservation is stored and then numeratorObservation. this is important for QDM execution results display

Reply:
No worries Prateek. Maybe `sortOberservations`? and one line documentation- sort observations to follow [denom observation, numer observation] order